source/mapping.d

```diff
@@ -42,17 +42,17 @@ M[VK_OEM_4]         = [mCH("dead_acute", '\u02ca'),     mCH("dead_tilde", '\u02d
 M['1']              = [mVK("1", '1'),                   mCH("degree", '°'),             mCH("onesuperior", '¹'),            mCH("ordfeminine", 'ª'),            mCH("onesubscript", '₁'),      mCH("notsign", '¬')];
 M['2']              = [mVK("2", '2'),                   mCH("section", '§'),            mCH("twosuperior", '²'),            mCH("masculine", 'º'),              mCH("twosubscript", '₂'),      mCH("logicalor", '∨')];
 M['3']              = [mVK("3", '3'),                   mCH("U2113", 'ℓ'),              mCH("threesuperior", '³'),          mCH("numerosign", '№'),             mCH("threesubscript", '₃'),    mCH("logicaland", '∧')];
-M['4']              = [mVK("4", '4'),                   mCH("guillemotright", '»'),     mCH("U203A", '›'),                  mCH("U22EE", '\u22ee'),             mCH("femalesymbol", '♀'),      mCH("uptack", '⊥')];
+M['4']              = [mVK("4", '4'),                   mCH("guillemotright", '»'),     mCH("U203A", '›'),                  VOID_KEY,                           mCH("femalesymbol", '♀'),      mCH("uptack", '⊥')];
```

Comment:
Nach Spec hast du völlig recht. NeoVars schreibt hier aber aus irgendeinem Grund genau dieses Zeichen, deswegen hatte ich den leeren Platz damit gefüllt. Kann der Korrektheit halber trotzdem weg.

Reply:
Das wollte ich nun genauer wissen und habe ein bisschen geforscht. In NeoVars ist das Zeichen seit der Neo2-Referenz (März 2010) auf der Vier.

Davor befand sich dort PageUp, welches nach der Neuabstimmung über den Navigationsblock auf X(4) rutschte – wo sich vorher eben genau „⋮“ befand. Das war noch die Anordnung von Neo2 RC1 von [Ende 2008](https://gitlab.freedesktop.org/xkeyboard-config/xkeyboard-config/-/blob/fbeda0e46d691aa1d956f3ef73151a5476fd0ef1/symbols/de). Da die Diskussionen über einen Ersatz auf 4(4) nichts ergeben haben, blieb der Platz leer bzw. wurde von mœsi damals im Ringtausch wieder von der vertikalen Ellipse belegt, [siehe hier](https://git.neo-layout.org/neo/neo-layout/commit/3c90146). Die Taste war eh frei, also hat es niemanden gestört … (Ende der Geschichtsstunde)

Ich würde das hier leer belassen, und dann kann sich jeder auch gerne selbst seinen Schneemann oder Totenkopf auf die Taste setzen.